src/caikit_nlp_client/__init__.py

```diff
@@ -1,5 +1,5 @@
 # noqa: F401
```

Comment:
Unrelated to this change, but we can get rid of this since it's not defined in `pyproject.toml`
```suggestion
```